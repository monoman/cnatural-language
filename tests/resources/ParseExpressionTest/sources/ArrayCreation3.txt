new T[1]{0}