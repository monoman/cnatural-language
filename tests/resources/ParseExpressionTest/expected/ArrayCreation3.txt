new T[Literal(DecimalInteger, [1])]{Literal(DecimalInteger, [0])}